docs/user-guide/attributes-table.md

```diff
@@ -113,10 +113,53 @@ In *Edit mode*, you can delete existing features following these steps:
 
 <img src="../img/delete-line-shape.gif" />
 
+Quick Filtering
+-------------------
+
+[MapStore](https://mapstore2.geo-solutions.it/mapstore/#/) attibute table allow you to quickly filtering the table records(features) by using the 
+filter field located at the top of each table column. 
+
+<p><img src="../img/attribute-table-quick-filter.png" style="max-width:620px;"/></p>
+
+To use attribute table quick filters, you simply have to type in the filter field with the filter data relate to the respected column. 
+
+* Forexample to filter **Arizona State** type in arizona in the filter field located on state_name column
+<p><img src="../img/attribute-table-quick-filter-1.png" style="max-width:620px;"/></p>
+
+!!! note
+    We have got 1 item and since the map is by default synchronized with the tabular data so all states have disappeared in the map except **Arizona** state.
+
+
+### Operator based filtering
+Quick filters allow you to do quick comparison operator based filtering by preceding the filter text with operator. 
```

Comment:
Quick filters allow you to filter records in the Attribute Table by using comparison operators (the operatos must proceed the filter text): operator can be used for all data type including float and date types.

Reply:
Sorry for my typo: **precede**